src/client.ts

```diff
@@ -503,6 +493,34 @@ export class Client extends EventEmitter {
     }
   }
 
+  /**
+   * It works by subscribing to particular events. The node will return a subscription id.
+   * For each event that matches the subscription a notification with relevant data is send
+   * together with the subscription id.
+   *
+   * @param method Method selected to the filter
+   * @param filter JSON string of the filter
+   */
+  async evmSubscribe(method: string, filter: string): Promise<string | null> {
```

Comment:
Should be `evmSubscribeAsync`, and can it really return `null`?

Can you make the doc a bit better, what is `method` here, contract method, or something else? What does a filter look like?